packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -138,12 +156,28 @@ export class AtomicNumericFacet
     );
   }
 
+  private areRangesEqual(
```

Comment:
Could exporting a `areRangesEqual` utility be useful for Headless?

Reply:
I don't think it's worthwhile and there's a discoverability problem. I expect most devs will write out the logic without looking for an export.